src/main/features/dashboard/views/macro/responseStatus.njk

```diff
@@ -42,11 +42,14 @@
   {% elseif claim.status === ClaimStatus.FREE_MEDIATION %}
     {{ t('{{ defendantName }} has rejected the claim. They’ve suggested free mediation on {{ respondedAt }}. You can accept or reject',
     { defendantName: claim.response.defendant.name, respondedAt: claim.respondedAt | date }) }}
+    <p><a href="{{ ResponsePaths.viewResponsePage.evaluateUri({ externalId: claim.externalId }) }}">{{ t('View full response') }}</a></p>
```

Comment:
Adding links here will mean that they will appear on the dashboard page as well. That is against of what I have heard that we should not put links into the main table. Probably something to do in the future however worth checking with UX whether they are happy to have these links for now till we refactor.